library/keysend.ts

```diff
@@ -52,14 +52,18 @@ async function constructKeysendMetadata(userId, externalKeysendRequest) {
 async function constructCustomRecords(keysend, keysendMetadata) {
   let customRecords = [];
   // Add standard values
-  customRecords.push({ type: 7629169, value: JSON.stringify(keysendMetadata) });
+  customRecords.push({
+    type: "7629169",
```

Comment:
for more context, i think it might be useful to name this type, looks like it comes from BLIP-0010? https://github.com/lightning/blips/blob/master/blip-0010.md
```suggestion
const BLIP0010 = "7629169"
  customRecords.push({
    type: BLIP0010,
```